Formula/pipx.rb

```diff
@@ -61,6 +61,23 @@ def install
     (bin/"pipx").write_env_script(libexec/"bin/pipx", PYTHONPATH: ENV["PYTHONPATH"])
     (bin/"register-python-argcomplete").write_env_script(libexec/"bin/register-python-argcomplete",
       PYTHONPATH: ENV["PYTHONPATH"])
+
+    # Install shell completions
+    output = Utils.safe_popen_read(libexec/"bin/register-python-argcomplete", "--shell=bash", "pipx")
+    (bash_completion/"pipx").write output
+
+    output = Utils.safe_popen_read(libexec/"bin/register-python-argcomplete", "--shell=fish", "pipx")
+    (fish_completion/"pipx.fish").write output
+
+    output = Utils.safe_popen_read(libexec/"bin/register-python-argcomplete", "--shell=bash", "pipx")
```

Comment:
```suggestion
    output = Utils.safe_popen_read(libexec/"bin/register-python-argcomplete", "--shell=zsh", "pipx")
```

Reply:
`zsh` actually does use `bash` completions, but I've removed it since it required manually enabling `bashcompinit`